recipe_scrapers/donnahay.py

```diff
@@ -8,23 +8,23 @@ class DonnaHay(AbstractScraper):
     def host(cls):
         return "donnahay.com.au"
 
-    def author(self):
-        return self.schema.author()
```

Comment:
Could probably keep this and have it return the site name based on the instructions from the scraper functions guide (https://github.com/hhursev/recipe-scrapers/blob/main/docs/in-depth-guide-scraper-functions.md)